choco/nunit-console-with-extensions.nuspec

```diff
@@ -33,5 +33,15 @@
 	<language>en-US</language>
     <tags>nunit console runner test testing tdd</tags>
     <copyright>Copyright (c) 2017 Charlie Poole, Rob Prouse</copyright>
+    <dependencies>
+      <group>
+          <dependency id="nunit-console-runner" version="$version$" />
+          <dependency id="nunit-extension-nunit-project-loader" version="3.5.0" />
+          <dependency id="nunit-extension-vs-project-loader" version="3.5.0" />
+          <dependency id="nunit-extension-nunit-v2-result-writer" version="3.5.0" />
+          <dependency id="nunit-extension-nunit-v2-driver" version="3.6.0" />
+        <dependency id="nunit-extension-teamcity-event-listener" version="1.0.2" />
```

Comment:
Actually, not so with the versions. Those listed are the last releases of each extension. The idea is that we bundle the last released extension version that we tested - in case they are released by other people.